api/src/pcapi/models/feature.py

```diff
@@ -148,26 +148,23 @@ def nameKey(self) -> str:
 
 FEATURES_DISABLED_BY_DEFAULT = (
     FeatureToggle.ALLOW_IDCHECK_REGISTRATION_FOR_EDUCONNECT_ELIGIBLE,
-    FeatureToggle.ALLOW_IDCHECK_UNDERAGE_REGISTRATION,
```

Comment:
pas trop d'accord avec le message de commit
*[PC-15780](https://passculture.atlassian.net/browse/PC-15780))[API] refactor: update Feature Flip default values on testing and staging envs*

-> pas besoin de mentionner testing et staging

Reply:
✅